packages/firebase_messaging/android/src/main/java/io/flutter/plugins/firebasemessaging/FirebaseMessagingPlugin.java

```diff
@@ -41,9 +41,14 @@
   public static void registerWith(Registrar registrar) {
     final MethodChannel channel =
         new MethodChannel(registrar.messenger(), "plugins.flutter.io/firebase_messaging");
+    final MethodChannel backgroundCallbackChannel =
+        new MethodChannel(registrar.messenger(), "plugins.flutter.io/android_fcm_background");
```

Comment:
Would the future iOS implementation not use this channel as well? Perhaps it should be called `plugins.flutter.io/firebase_messaging_background`?

Reply:
Done